backend/app/core/dependencies.py

```
@@ -0,0 +1,92 @@
+        raise AuthenticationError("유효하지 않은 토큰입니다.")
+
+    user_id = payload.get("sub")
+    if not user_id:
+        raise AuthenticationError("토큰에 사용자 정보가 없습니다.")
+
+    return user_id
+
+
+def get_current_user(
+    user_id: str = Depends(get_current_user_id),
+    db: Session = Depends(get_db)
```

Ruff (0.14.5)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)